test_scripts/Polices/External_UCS/External_Consent_OFF/030_ATF_Policies_External_Consent_OFF_empty_consent_groups.lua

```diff
@@ -0,0 +1,173 @@
+--------------------------------------Requirement summary---------------------------------------------
+-- [Policies] External UCS: empty "consent_groups" changes after externalConsentStatus was received
+
+------------------------------------General Settings for Configuration--------------------------------
+require('user_modules/all_common_modules')
+local common_functions_external_consent = require('user_modules/ATF_Policies_External_Consent_common_functions')
+local common_steps = require('user_modules/common_steps')
+local common_functions = require ('user_modules/common_functions')
```

Comment:
@maria-mih, Explicit defining of 'common_steps' and 'common_functions' is not required since both libraries are defined in 'all_common_modules'